offices/_single.html

```diff
@@ -1,8 +1,11 @@
 {% extends "layout-side-nav.html" %}
 {% import "_vars-offices.html" as vars with context %}
 
+{% set office_title = office.title | safe if office.title %}
```

Comment:
Why define these instead of just grabbing the object later in the template? Also, what happens if there's no title or tags? Shouldn't those be conditioned where they're used or else be blank?